content/en/templates/shortcode.md

```diff
@@ -39,10 +39,14 @@ Note the forward slash.
 
 ### Shortcode template lookup order
 
-Shortcode templates have a simple [lookup order]:
+Shortcode templates have a simple [lookup order], which first looks for a shortcode unique to the [output format] and then falls back to a generic HTML version.
 
-1. `/layouts/shortcodes/<SHORTCODE>.html`
-2. `/themes/<THEME>/layouts/shortcodes/<SHORTCODE>.html`
+1. `/layouts/shortcodes/<SHORTCODE>.<OUTPUT-BASENAME>.<OUTPUT-SUFFIX>`
+2. `/themes/<THEME>/layouts/shortcodes/<OUTPUT-BASENAME>.<OUTPUT-SUFFIX>`
+3. `/layouts/shortcodes/<SHORTCODE>.html`
+4. `/themes/<THEME>/layouts/shortcodes/<SHORTCODE>.html`
+
+For example, the built-in RSS output format uses a baseName of `index` and suffix of `xml` by default. When calling a shortcut via `{{</* example */>}}`, the RSS template will first look for a file `example.index.xml` and if one doesn't exist will fall back to `example.html`.
```

Comment:
I'm pretty sure the above isn't correct: We use the output `name`,  eg. `rss`, so `foo.rss.xml` should work.

Reply:
Ahhh, good catch. I tested this, but failed to remember that my config is custom in this area, specifically I change the RSS output basename to "rss", which confused me. Fixed.